native/src/main/java/io/ballerina/stdlib/java/jdbc/Constants.java

```diff
@@ -0,0 +1,200 @@
+        public static final BString FIREWALL_SERVER = StringUtils.fromString("FirewallServer");
+        public static final BString FIREWALL_PORT = StringUtils.fromString("FirewallPort");
+        public static final BString FIREWALL_USER = StringUtils.fromString("FirewallUser");
+        public static final BString FIREWALL_PASSWORD = StringUtils.fromString("FirewallPassword");
+        // Proxy
+        public static final BString PROXY_AUTO_DETECT = StringUtils.fromString("ProxyAutoDetect");
+        public static final BString PROXY_SERVER = StringUtils.fromString("ProxyServer");
+        public static final BString PROXY_PORT = StringUtils.fromString("ProxyPort");
+        public static final BString PROXY_AUTH_SCHEMA = StringUtils.fromString("ProxyAuthScheme");
+        public static final BString PROXY_USER = StringUtils.fromString("ProxyUser");
+        public static final BString PROXY_PASSWORD = StringUtils.fromString("ProxyPassword");
+        public static final BString PROXY_SSL_TYPE = StringUtils.fromString("ProxySSLType");
+        public static final BString PROXY_EXCEPTIONS = StringUtils.fromString("ProxyExceptions");
+        // Logging
+        public static final BString LOG_FILE = StringUtils.fromString("Logfile");
+        public static final BString VERBOSITY = StringUtils.fromString("Verbosity");
+        public static final BString LOG_MODULES = StringUtils.fromString("LogModules");
+        public static final BString MAX_LOG_FILE_SIZE = StringUtils.fromString("MaxLogFileSize");
+        public static final BString MAX_LOG_FILE_COUNT = StringUtils.fromString("MaxLogFileCount");
+        // Miscellaneous
+        public static final BString BATCH_SIZE = StringUtils.fromString("BatchSize");
+        public static final BString CONNECTION_LIFE_TIME = StringUtils.fromString("ConnectionLifeTime");
+        public static final BString CONNECT_ON_OPEN = StringUtils.fromString("ConnectOnOpen");
+        public static final BString MAX_ROWS = StringUtils.fromString("MaxRows");
+        public static final BString OTHER = StringUtils.fromString("Other");
+        public static final BString POOL_IDLE_TIMEOUT = StringUtils.fromString("PoolIdleTimeout");
+        public static final BString POOL_MAX_SIZE = StringUtils.fromString("PoolMaxSize");
+        public static final BString POOL_MIN_SIZE = StringUtils.fromString("PoolMinSize");
+        public static final BString POOL_WAIT_TIME = StringUtils.fromString("PoolWaitTime");
+        public static final BString PSEUDO_COLUMNS = StringUtils.fromString("PseudoColumns");
+        public static final BString QUERY_PASS_THROUGH = StringUtils.fromString("QueryPassthrough");
+        public static final BString RTK = StringUtils.fromString("RTK");
+        public static final BString TIMEOUT = StringUtils.fromString("Timeout");
+        public static final BString USE_CONNECTION_POOLING = StringUtils.fromString("UseConnectionPooling");
```

Comment:
Please remove these unused constants from the CData connect module.

Reply:
Removed that constants